OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -122,7 +122,10 @@ public MainMenuLogic(Widget widget, World world, ModData modData)
 
 			missionsButton.Disabled = !hasCampaign && !hasMissions;
 
-			singleplayerMenu.Get<ButtonWidget>("SKIRMISH_BUTTON").OnClick = StartSkirmishGame;
+			var hasMaps = modData.MapCache.Any(p => !p.Visibility.HasFlag(MapVisibility.Shellmap));
```

Comment:
These should instead be checking `.Any(p => p.Visibility.HasFlag(MapVisibility.Lobby))`, which is the criteria used in the map chooser.